cli/atlas/templates/pkg/pb/service.proto.gotmpl

```diff
@@ -5,7 +5,8 @@ package service;
 import "google/protobuf/empty.proto";
 import "google/api/annotations.proto";
 import "github.com/lyft/protoc-gen-validate/validate/validate.proto";
-import "protoc-gen-swagger/options/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";{{ if .WithDatabase }}
```

Comment:
nitpick: I feel like these templates would all be cleaner if the whole conditional statement was on one line... why does it need to be split across two lines like this?